frontend/src/components/quiz/QuizQuestion.tsx

```diff
@@ -1,4 +1,4 @@
-import { Audio, Image } from "../general";
+import { Audio, Image } from "../general/general";
```

Comment:
```suggestion
import { Audio, Image } from "@/components/general/general";
```

지금 하지 말고 추후에 한꺼번에 바꿀 것, 포맷팅 도구에서 할 수 있는 방법이 있는지 알아보기